src/openzaak/components/zaken/api/viewsets.py

```diff
@@ -1440,6 +1441,40 @@ def perform_destroy(self, instance):
         },
     )
 )
+@extend_schema_view(
```

Comment:
Above this, the same decorator `extend_schema_view` is applied, so these sort of overlap. Could you make sure this decorator is used only once and add your changes to the existing decorator?